### PR TITLE
Feature/adding links of internal pages

### PR DESCRIPTION
**Pull request by jonataspinto**
## Description
Com a remoção da seção de inscreva-se estávamos sem link para `perguntas-frequentes`. Nesse pr adiciono o mesmo link no footer e link para home nas logos.

## Screenshots / Videos
image

image
## Developer Checks

- [x] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [x] Tested on browser device
- [x] Tested on Mobile device

### CODE REVIEW

**Comment by estartandodevs** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.84 | ✅ |
| First Contentful Paint | 1.6 s |  |
| Time to Interactive | 7.8 s |  |
| Speed Index | 3.1 s |  |
| Total Blocking Time | 330 ms |  |
| Largest Contentful Paint | 2.5 s |  |
| Cumulative Layout Shift | 0.021 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.8 | ✅ |
| First Contentful Paint | 2.7 s |  |
| Time to Interactive | 7.2 s |  |
| Speed Index | 4.0 s |  |
| Total Blocking Time | 170 ms |  |
| Largest Contentful Paint | 3.2 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

File: src/components/CommonQuestions/index.tsx [hunk 1]
```diff
@@ -15,7 +16,11 @@ export type CommonQuestionsProps = {
 export const CommonQuestions = ({ title, questions }: CommonQuestionsProps) => (
   <S.CommonQuestionsSection>
     <header>
-      <Logo width={48} height={68} priority quality={30} />
+      <Link href="/">
+        <a>
```
Comment (ramonxm) [inline src/components/CommonQuestions/index.tsx:20]: Nesse caso não precisa da tag `a` visto que não ta utilizando o `_blank`,  correto?

**Review by estartandodevs — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

File: src/components/Footer/index.tsx [hunk 2]
```diff
@@ -4,18 +4,29 @@ import * as S from './styles';
...
 
 import { Logo } from '../Logo';
+import Link from 'next/link';
 
 export const Footer = () => (
   <S.FooterWrapper>
     <S.FooterContainer>
-      <Logo width={65} height={94} />
+      <Link href="/">
+        <a>
```
Comment (ramonxm) [inline src/components/Footer/index.tsx:13]: Mesma coisa.

**Review by ramonxm — CHANGES_REQUESTED** [review]

File: src/components/CommonQuestions/index.tsx (hunk 1 above)
Comment (jonataspinto) [reply src/components/CommonQuestions/index.tsx:20]: Infelizmente sim, olha os erros ao tentar usar sem.

https://nextjs.org/docs/messages/link-passhref

image

image

Comment (ramonxm) [reply src/components/CommonQuestions/index.tsx:20]: I see, seguimos.

**Review by ramonxm — APPROVED** [review]
LGTM

**Comment by estartandodevs** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.67 | ✅ |
| First Contentful Paint | 1.2 s |  |
| Time to Interactive | 7.8 s |  |
| Speed Index | 3.8 s |  |
| Total Blocking Time | 420 ms |  |
| Largest Contentful Paint | 4.6 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.8 | ✅ |
| First Contentful Paint | 2.6 s |  |
| Time to Interactive | 7.1 s |  |
| Speed Index | 4.1 s |  |
| Total Blocking Time | 170 ms |  |
| Largest Contentful Paint | 3.2 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

**Review by estartandodevs — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by estartandodevs** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.33 | ✅ |
| First Contentful Paint | 3.1 s |  |
| Time to Interactive | 9.2 s |  |
| Speed Index | 6.1 s |  |
| Total Blocking Time | 1,260 ms |  |
| Largest Contentful Paint | 6.2 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.86 | ✅ |
| First Contentful Paint | 2.4 s |  |
| Time to Interactive | 5.7 s |  |
| Speed Index | 3.6 s |  |
| Total Blocking Time | 110 ms |  |
| Largest Contentful Paint | 3.1 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

**Review by estartandodevs — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Pull request merged**